androidHyperskillApp/src/main/res/layout/item_track.xml

```diff
@@ -71,8 +90,10 @@
             app:constraint_referenced_ids="trackSelectedBadge,trackCompletedBadge,trackBetaBadge"
             android:paddingTop="@dimen/project_selection_list_item_tags_top_margin"
             app:layout_constraintTop_toBottomOf="@+id/trackTimeToCompleteTextView"
-            app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="@id/startGuideline"
+            app:layout_constraintEnd_toEndOf="@id/endGuideline"
+            android:paddingHorizontal="@dimen/item_horizontal_padding"
+            android:paddingBottom="@dimen/item_vertical_padding"
             />
```

Comment:
A view should use `android:layout_width="0dp"` to be constrained, according to the `constraintLayout` rules. Otherwise, it will occupy all the available width.

Reply:
@XanderZhu where can I read more about such things, please, suggest me some sources 🙏